openupgrade_scripts/scripts/website/14.0.1.0/pre-migration.py

```diff
@@ -39,6 +43,20 @@ def website_cookie_notice_pre_migration(env):
     )
 
 
+def convert_html_string(env):
+    env.cr.execute(
+        "SELECT * FROM ir_ui_view WHERE type = 'qweb' AND website_id IS NOT NULL"
+    )
+    website_views = list(chain.from_iterable(env.cr.fetchall()))
+    for view in website_views:
+        new_arch = convert_html_string_13to14(view.arch_db)
+        view.arch_db = new_arch
+        openupgrade.logged_query(
+            env.cr,
+            f"UPDATE ir_ui_view SET arch_db = {view.arch_db} WHERE id = {view.id}",
+        )
```

Comment:
```suggestion
        view.arch_db = convert_html_string_13to14(view.arch_db)
```

Reply:
Done! but I have a doubt: In this way it is not necessary to execute the write to update the database?
I have included it because I think so, correct me if I am wrong.